drivers/flash/flash_stm32_xspi.c

```diff
@@ -2121,11 +2127,13 @@ static int flash_stm32_xspi_init(const struct device *dev)
 	} else {
 
 	}
+#if !defined(CONFIG_SOC_SERIES_STM32N6X)
 #if STM32_XSPI_DLYB_BYPASSED
```

Comment:
could the bit  field OCTOSPI_DCR1_DLYBYP of the DCR1 register by used there ?

Reply:
I think XSPI_DCR1_DLYBYP is better suited here, replaced with that